index.html

```diff
@@ -86,6 +86,14 @@ <h4 class="modal-title">Extra Settings</h4>
               <input type="text" class="form-control input-md" id="request-delay" placeholder="0"  rv-value="request_delay">
               <p class="help-block">(milliseconds, default 0) how long to wait between making requests</p>
             </div>
+            <div class="form-group form-inline">
+              <input type="checkbox" class="form-control" name="lastfm-include-headers" id="lastfm-include-headers" checked="false">
```

Comment:
(just a suggestion, feel free to ignore)

It may be possible to use [Rivets rv-checked](http://rivetsjs.com/docs/reference/#checked), to automatically bind this value. 

```suggestion
              <input type="checkbox" class="form-control" name="lastfm-include-headers" rv-checked="include_headers">
```

Then in the code later you could access this value through the state:

```js
var includeHeaders = state.include_headers;
```

……… However, I've no idea what version of Rivets this is using, and if there are any issues with the different options, so feel free to ignore this and I could take a look later.